### PR TITLE
Do not hard-code task name in qcRunReadout

### PR DESCRIPTION
**Pull request by Barthelemy**
The aim is to be able to rename easily the task in the config file.

### CODE REVIEW

File: Framework/src/runnerUtils.h [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+{
+
+/**
+ * Returns the name of the first task encountered in the config file.
+ * Ad-hoc solution to avoid hard-coding the task when we create the printer (he needs it to know the data description
+ * of the data coming out of the checker).
+ * @param config
+ * @return The name of the first task in the config file.
+ */
+std::string getFirstTaskName(std::string configurationSource)
```
Comment (knopers8) [inline Framework/src/runnerUtils.h]: Nice idea, much more convenient for small developments.

**Comment by knopers8** [comment]
Travis seems to be a bit uncommunicative about the problem with runnerUtils.h
"error: `/usr/bin/clang-format-7 -lines=1:33 Framework/src/runnerUtils.h` failed"

File: Framework/src/runnerUtils.h [hunk 2]
```diff
@@ -0,0 +1,33 @@
+//
+// Created by bvonhall on 16/04/19.
+//
```
Comment (knopers8) [inline Framework/src/runnerUtils.h]: Could you add the license notice?

Comment (Barthelemy) [reply Framework/src/runnerUtils.h]: done

**Comment by Barthelemy** [comment]
not sure what is the problem of travis.

**Comment by knopers8** [comment]
@awegrzyn Travis still reports the same problem after bumping clang-format. Could you have a look?

**Comment by awegrzyn** [comment]
Ok, where is actually `Framework/src/runnerUtils.h`? Can't find it in the repo.

**Comment by knopers8** [comment]
It is a new file, you can find it in the PR

**Comment by knopers8** [comment]
It actually got weirder after https://github.com/AliceO2Group/QualityControl/pull/142 :
```
Configuration file(s) do(es) not support C++: /home/travis/build/AliceO2Group/QualityControl/.clang-format
error: `/usr/bin/clang-format-8 -lines=65:65 -lines=72:72 -lines=101:102 Framework/src/runReadout.cxx` failed
```

File: Framework/src/Quality.cxx [hunk 3]
```diff
@@ -7,9 +7,10 @@
 
 ClassImp(o2::quality_control::core::Quality)
 
+  // clang-format off
```
Comment (awegrzyn) [inline Framework/src/Quality.cxx]: I guess we should modify clang-format to accept that

Comment (knopers8) [reply Framework/src/Quality.cxx]: If you mean the clang-format off/on commands - it accepts them already. 
If you mean moving this line two spaces to the right, this seems to be a bug connected somehow to the usage of the macro above. Do you know to deal with it properly?

Comment (Barthelemy) [reply Framework/src/Quality.cxx]: I had tested the new namespace style with clang-format and it worked. Is it actually not working here ?

Comment (knopers8) [reply Framework/src/Quality.cxx]: for me it was suggesting to indent it with two spaces

Comment (Barthelemy) [reply Framework/src/Quality.cxx]: ok, understood

**Pull request merged**